packages/app-root/src/app/groups/[groupId]/page.js

```diff
@@ -1,12 +1,13 @@
-'use client'
+import GroupStatsContainer from './GroupStatsContainer'
 
-import { GroupStats } from '@zooniverse/user'
-import auth from 'panoptes-client/lib/auth'
+export const metadata = {
+  title: 'Group Stats',
```

Comment:
Not necessarily required for Core v1, but is there any way this title can include the `groupId`? Otherwise all individual group stats pages will have the exact same title. If I find docs on how to do so server side, I'll link it here.